### PR TITLE
feat(chat): members availability and settings change

### LINKED ISSUE #4906: update app defaults
### Description

- [x] `Experimental features -> Gif Widget` should be enabled by default
- [x] `Experimental features -> Online users` should be enabled by default
- [x] remove toggle `Experimental features -> Gif Widget`
- [x] remove toggle `Experimental features -> Online users`
- [x] remove/hide online users toggle from public channels

### PR DESCRIPTION
**Pull request by stefandunca**
### Fixes #4906

Don't show members in public chat
Show members in communities by default
Enable Gif Widget by default and remove settings entry
Enable Online users by default and remove settings entry

Additional qml error fix: 511f7bb8

### Affected areas

Settings, Communities and Public chats

### Screenshot of functionality

misisng_members_for_public

first_time_comunities_experience

no_more_public_and_gifs

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatLayout.qml [hunk 1]
```diff
@@ -81,10 +81,11 @@ StatusAppThreePanelLayout {
     }
 
     showRightPanel: {
-        if (!localAccountSensitiveSettings.showOnlineUsers || !localAccountSensitiveSettings.expandUsersList) {
+        let chatContentModule = root.rootStore.currentChatContentModule()
+        if (!localAccountSensitiveSettings.showOnlineUsers || !localAccountSensitiveSettings.expandUsersList
+                || (chatContentModule && chatContentModule.chatDetails.type === Constants.chatType.publicChat)) {
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/ChatLayout.qml:86]: Just a suggestion.. Could we manage `chatContentModule` conditions in a separate `if` condition maybe grouping it with the next condition in line 88/89?

**Review by noeliaSD — APPROVED** [review]
LGTM! :)

**Review by osmaczko — APPROVED** [review]

File: ui/app/AppLayouts/Chat/ChatLayout.qml (hunk 1 above)
Comment (stefandunca) [reply ui/app/AppLayouts/Chat/ChatLayout.qml]: Done, thanks @noeliaSD for the suggestion

**Review by iurimatias — APPROVED** [review]
good job

**Comment by iurimatias** [comment]
tested requirements in the original issue

**Pull request merged**